lib/templates/erb/scaffold/_form.html.erb

```diff
@@ -0,0 +1,14 @@
+<%%= simple_form_for(@<%= singular_table_name %>) do |f| %>
```

Comment:
what's this?

Reply:
the simple form installation added this file 🤔 , im going to remove it !